bndtools.core.test/src/bndtools/core/test/editors/quickfix/AbstractBuildpathQuickFixProcessorTest.java

```diff
@@ -331,7 +332,8 @@ void before() throws Exception {
 	private static final String CLASS_FOOTER = " var};";
 	protected static final Set<Integer> SUPPORTED = Stream
```

Comment:
This should probably be `Sets.of`. This uses the immutable collections support in aQute.libg.

Reply:
Done.